src/renderer/corporate/common/common-helpers.jsx

```diff
@@ -34,9 +34,9 @@ const getEntityJurisdiction = entry => {
 		entry.identity.type === 'individual'
 			? 'http://platform.selfkey.org/schema/attribute/nationality.json'
 			: 'http://platform.selfkey.org/schema/attribute/legal-jurisdiction.json';
-	const attribute = entry.attributes.find(a => a.type.content.$id === idAttribute);
```

Comment:
I would propose to not do attribute searches like that in container in general, this should be inside profile selector

Reply:
Agree 💯 
Please review this later in the edit-members pr.